### ISSUE TITLE
Anonymize user names/emails/etc

### ISSUE THREAD

**Issue by ryan953** [getsentry/sentry#38207]
We can stomp on the email field and just not show it, which would be a quick front-end change (but would apply to every org)

Followup questions pop up though:
1. Should we continue to include email in search?
   - The product should include email, it's really useful if someone gets a support ticket or something, to lookup a replay by the user who submitted the ticket
2. Is showing `user_id` ok?
    - You can still get the email address by doing a disco query
    - granularity of this change, should we show userid's?
3. Is `user` important to show at all?
    - searching for support tickets is a use-case
    - showing browser/os might be more interesting sometimes when debugging
4. What if names/emails are gone from the list endpoint, but appear in details
    - not as creepy on the list page
    - full utility on details page
5. PII on `/_admin` and when logged in as `su`  is more important
6. Change is easier once `display_name` field is merged into API backend (work already started)
    - Done now: https://github.com/getsentry/replay-backend/pull/104/files

**Comment by ryan953** [comment]
This is superseded by the PII scrubbing backend work. Closing.
